install/installer/pkg/common/render_test.go

```diff
@@ -28,6 +29,46 @@ func TestCompositeRenderFunc_NilObjectsNilError(t *testing.T) {
 	require.Len(t, objects, 0)
 }
 
+func TestReplicas(t *testing.T) {
+	testCases := []struct {
+		Component        string
+		ExpectedReplicas int32
+	}{
+		{
+			Component:        "server",
+			ExpectedReplicas: 123,
+		},
+		{
+			Component:        "dashboard",
+			ExpectedReplicas: 456,
+		},
+		{
+			Component:        "content-service",
```

Comment:
[recommendation]: Give each test a name as a string, like 
```
name: "defaults to 1 for components without experimental pod config"
```

You can then even do 
```
for _, scenario := range scenarios {
  t.Run(scenario.name, func(t *testing.T) {
     // your test logic
  }))
}
```
Which prints nice errors when a particular table test fails

Reply:
Done in dedf7d3d53e366f93e01a0c804450bab96d8ee19